src/mesh/wifi/WiFiAPClient.cpp

```diff
@@ -92,11 +96,13 @@ static void onNetworkConnected()
             syslog.enable();
         }
 
+#if !MESHTASTIC_EXCLUDE_WEBSERVER
+
 #ifdef ARCH_ESP32
         initWebServer();
 #endif
         initApiServer();
-
+#endif
```

Comment:
That technically works, but it excludes the API Socket Server which may be available on other platforms (nRF52 with ETH) if that macro is set there errorneously. For clarity, can you add an ELSE branch to the outer macro making sure the apiserver is initialized even when the macro is set?

Reply:
Hi, thank you for the comments. I had done as you suggest initially, but in testing I realized that the API server relies on the http(s) server anyway and is disabled by this change. There might be some utility in separating the two services, but it won't create any substantial space savings. For platforms with ETH, I think that initAPIServer() will be called from ethClient.ccp (and will probably fail when webserver is disabled, now that I think of it).

Reply:
OK, I see what I was missing before. There is an API server running on a separate port, independent of the web/http servers. Adding an update to the PR.